### DISCUSSION TITLE
upgrade to 1.18?

### DISCUSSION THREAD

**Discussion by skicek** [MrTJP/ProjectRed#1711] [Q&A] [answered]
Hello, do you have plans to upgrade project red to 1.18? or you will wait till 1.19?

**Comment by MrTJP** [comment, marked as answer]
Hi. I believe our core libraries are migrating to 1.18 as we speak. That is where ProjectRed will land next.

**Reply by MadDog443** [reply]
Apologies as I'm not trying to rush but do you have any ETA on plans to port to newer versions? Currently working on a modpack for a fairly large private server and it's hard to figure out the ideal MC version to run when it comes to compatibility cause regenerating the map isn't possible when porting forward... Thanks.

**Reply by MrTJP** [reply]
Just finished the port few days ago. Needs some minor bug fixing but hoping to release it by this weekend, then move it to 1.19+ right after

**Reply by MadDog443** [reply]
Amazing, thank you for the update, your mods will make for a great addition!

**Comment by MrTJP** [comment]
FYI, we released 1.18 quite a while ago (https://github.com/MrTJP/ProjectRed/discussions/1775). 1.19 is in beta, and 1.20 is complete and pending some dependency releases. Closing this discussion..
